src/ast/types/blocks-and-statements.ts

```diff
@@ -69,8 +79,18 @@ export interface VariableDeclarator {
 export type VariableDeclaratorId = Identifier;
 export type VariableInitializer = Expression;
 
-export type Expression = Primary | BinaryExpression | UnaryExpression;
-export type Primary = Literal | ExpressionName | Assignment;
+export interface ReturnStatement {
+  kind: "ReturnStatement";
+  exp: Expression;
+}
+
+export type Expression = Primary | BinaryExpression | UnaryExpression | Void;
+
+export interface Void {
+  kind: "Void";
+}
```

Comment:
By `Void`, do you mean the `void` return type in Java? I think it should be under `MethodDeclaration -> MethodHeader -> Result` instead of `Expression`.

Reply:
No, `Void` here refers to "nothing" which is returned by methods declared `void`.

I admit this is a hack to facilitate ECE evaluation as `Void` is required to be a `Node` as well.

Reply:
On second thought, maybe this change is too much. I shall keep this in mind and perhaps propose an alternate solution that make modifications in the ECE instead in the future.